test/test_jld2_output_writer.jl

```diff
@@ -88,6 +88,52 @@ function test_jld2_file_splitting(arch)
     return nothing
 end
 
+function test_jld2_file_splitting_time(arch)
+    grid = RectilinearGrid(arch, size=(16, 16, 16), extent=(1, 1, 1), halo=(1, 1, 1))
+    model = NonhydrostaticModel(; grid, buoyancy=SeawaterBuoyancy(), tracers=(:T, :S))
+    simulation = Simulation(model, Δt=1, stop_iteration=10)
+
+    function fake_bc_init(file, model)
+        file["boundary_conditions/fake"] = π
+    end
+    ow = JLD2OutputWriter(model, (; u=model.velocities.u);
+                          dir = ".",
+                          filename = "test.jld2",
+                          schedule = IterationInterval(1),
+                          init = fake_bc_init,
+                          including = [:grid],
+                          array_type = Array{Float64},
+                          with_halos = true,
+                          file_splitting = FileTimeSplit(3seconds),
```

Comment:
```suggestion
                          file_splitting = TimeInterval(3seconds),
```

start with this change then we can see the errors